### ISSUE TITLE
Error while exporting to html

### ISSUE THREAD

**Issue by fcui** [kirbs-/hide_code#53]
Hi, 

I have installed hide_code 0.5. While exporting to html, I met an error as the following:
"500 : illegal multibyte sequence".
In the notebook, there are Chinese characters.

I'd really appreciate some help.

**Comment by kirbs-** [comment]
I'll check. There might be some character encoding at play here.

**Comment by xxxspy** [comment]
Error happened  when uri contain non-latin-1 characters: 
```
[I 11:24:00.539 NotebookApp] hide_code: Starting HTML export for /test富文本编辑器.ipynb
[E 11:24:00.724 NotebookApp] Uncaught exception GET /notebooks/learnings/testNotebook/test%E5%AF%8C%E6%96%87%E6%9C%AC%E7%BC%96%E8%BE%91%E5%99%A8.ipynb/export/html (172.16.160.216)
    HTTPServerRequest(protocol='http', host='172.16.160.203:8880', method='GET', uri='/notebooks/learnings/testNotebook/test%E5%AF%8C%E6%96%87%E6%9C%AC%E7%BC%96%E8%BE%91%E5%99%A8.ipynb/export/html', version='HTTP/1.1', remote_ip='172.16.160.216')
    Traceback (most recent call last):
      File "/home/bnu/syd/jupyterNotebook/.env/lib/python3.6/site-packages/tornado/web.py", line 1541, in _execute
        result = method(*self.path_args, **self.path_kwargs)
      File "/home/bnu/syd/jupyterNotebook/.env/lib/python3.6/site-packages/hide_code/hide_code.py", line 31, in get
        self.flush()
      File "/home/bnu/syd/jupyterNotebook/.env/lib/python3.6/site-packages/tornado/web.py", line 973, in flush
        start_line, self._headers, chunk, callback=callback)
      File "/home/bnu/syd/jupyterNotebook/.env/lib/python3.6/site-packages/tornado/http1connection.py", line 385, in write_headers
        lines.extend(l.encode('latin1') for l in header_lines)
      File "/home/bnu/syd/jupyterNotebook/.env/lib/python3.6/site-packages/tornado/http1connection.py", line 385, in <genexpr>
        lines.extend(l.encode('latin1') for l in header_lines)
    UnicodeEncodeError: 'latin-1' codec can't encode characters in position 46-51: ordinal not in range(256)
```

**Comment by kirbs-** [comment]
This is resolved in v0.5.3.
